Specs/DateTime/Dutch/DurationParser.json

```diff
@@ -595,43 +573,41 @@
         "Start": 7,
         "Length": 13
       }
-    ],
-    "NotSupported": "dotNet",
-    "NotSupportedByDesign": "javascript,python,java"
+    ]
   },
   {
     "Input": "Ik ben anderhalve dag weg",
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
+    "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
         "Text": "anderhalve dag",
         "Type": "duration",
         "Value": {
           "Timex": "P1.5D",
           "FutureResolution": {
-            "duration": "7776000"
+            "duration": "129600"
           },
           "PastResolution": {
-            "duration": "7776000"
+            "duration": "129600"
           }
         },
         "Start": 7,
         "Length": 14
       }
-    ],
-    "NotSupported": "dotNet",
-    "NotSupportedByDesign": "javascript,python,java"
+    ]
   },
   {
     "Input": "Ik ben een kwartier uur weg",
```

Comment:
This spec was incorrect, sorry. It could be `een kwartier` or `een kwart uur`.

Reply:
updated the test case to "Ik ben een kwartier weg"